app.json

```diff
@@ -28,6 +28,14 @@
       "description": "Payment Orchestration environment to connect to.",
       "required": true,
       "value": "payments.sandbox.verygoodsecurity.app"
+    },
+    "ROUTE_ID": {
+      "description": "Route id from dashboard",
```

Comment:
Route ID for which route? Please change description to include the name of the route so it's easy to find in the dashboard.

Reply:
And if the ROUTE_ID will never change, let's just hard-code it, rather than making it configurable.